<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-KTHKFE5" name="tests.login_test" tests="9" failures="1" timestamp="2020-11-29T16:32:56 IST" time="11.319" errors="0">
  <testcase name="t01_loginFailedFromTable" time="1.354" classname="tests.login_test"/>
  <system-out/>
  <testcase name="t01_loginFailedFromTable" time="1.364" classname="tests.login_test"/>
  <system-out/>
  <testcase name="t02_loginFailedMatchAssert" time="1.352" classname="tests.login_test"/>
  <system-out/>
  <testcase name="t03_loginFailedNoMatchAssert" time="1.352" classname="tests.login_test">
    <failure type="java.lang.AssertionError" message="expected [Sorry! Epic sadface: Username and password do not match any user in this service] but found [Epic sadface: Username and password do not match any user in this service]">
      <![CDATA[java.lang.AssertionError: expected [Sorry! Epic sadface: Username and password do not match any user in this service] but found [Epic sadface: Username and password do not match any user in this service]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at tests.login_test.t03_loginFailedNoMatchAssert(login_test.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
    </failure>
  </testcase> <!-- t03_loginFailedNoMatchAssert -->
  <system-out/>
  <testcase name="getResponseTime" time="0.481" classname="tests.login_test"/>
  <system-out/>
  <testcase name="t01_loginFailedFromTable" time="1.422" classname="tests.login_test"/>
  <system-out/>
  <testcase name="t01_loginFailedFromTable" time="1.363" classname="tests.login_test"/>
  <system-out/>
  <testcase name="t04_loginSucceed" time="2.131" classname="tests.login_test"/>
  <system-out/>
  <testcase name="getStatusCode" time="0.500" classname="tests.login_test"/>
  <system-out/>
</testsuite> <!-- tests.login_test -->
